flathub.json

```
@@ -0,0 +1,3 @@
+{
+  "only-arches": ["i386", "x86_64"]
```

```suggestion
  "only-arches": ["x86_64"]
```

We don't support i386 architectures